server/client.go

```diff
@@ -195,6 +196,15 @@ func (client JiraClient) GetIssue(key string, options *jira.GetQueryOptions) (*j
 	return issue, nil
 }
 
+// GetWatchers returns an Issue watchers by issueKey.
```

Comment:
```suggestion
// GetWatchers returns an array of Jira watchers for a given issue.
```

Reply:
@jfrerich I've pushed a new commit for this change, could you please check it?